packages/ui/src/theme/tokens/types/designToken.ts

```diff
@@ -1,20 +1,5 @@
 /**
- * Helper function to test if something is a design token or not.
- * Used in the React component style props.
- *
- * @param arg - thing to test if it is a design token or not
- * @returns boolean
- */
-export function isDesignToken(arg: unknown): arg is WebDesignToken {
```

Comment:
Moved this to _tokens/utils.ts_ in an effort to begin reducing the amount of non-TS that is inside of types files